docs/_optimist/storage/localstorage/index.en.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Localstorage
+lang: en
+permalink: /optimist/storage/localstorage/
+nav_order: 3200
+parent: storage
```

Comment:
Capitalize! Needs to match the nav entry otherwise it won't be shown (as currently)

Reply:
done